src/components/layout/nav.js

```diff
@@ -120,8 +120,14 @@ const NavCenter = styled.ul`
     display: flex;
     justify-content: space-between;
 
+    @media (max-width: 1204px) {
+        font-size: 14px;
+    }
+    @media (max-width: 1175px) {
+        font-size: 12px;
+    }
     @media (max-width: 1104px) {
-        font-size: var(--text-size-xs);
+        font-size: 10px;
```

Comment:
please make the smallest font-size to 12px, otherwise in desktop it will be very small, instead for 1175, you can try to decrease logo size